cmake/ConfigureInstall.cmake

```diff
@@ -14,14 +14,7 @@ list(APPEND CMAKE_PREFIX_PATH "${GENNY_INSTALL_DIR}")
 
 include(GNUInstallDirs)
 
-# This line allows us to run genny executables from a distribution folder where libraries live in
-# bin/../lib64. This is a nice thing for driver-like applications, because you can pass around
-# archives trivially.
-set(CMAKE_INSTALL_RPATH                 "\$ORIGIN/../${CMAKE_INSTALL_LIBDIR}"   CACHE PATH "")
-
-# Since we're shooting for redistribution, we change up the RPATH configuration
-set(CMAKE_SKIP_BUILD_RPATH              false   CACHE BOOL "")
-set(CMAKE_SKIP_INSTALL_RPATH            false   CACHE BOOL "")
-set(CMAKE_BUILD_WITH_INSTALL_RPATH      false   CACHE BOOL "")
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH   true    CACHE BOOL "")
-set(CMAKE_MACOSX_RPATH                  ON      CACHE BOOL "Ensure that RPATH is used on OSX")
+set(CMAKE_MACOSX_RPATH                  true                          CACHE BOOL "Ensure that RPATH is used on OSX")
```

Comment:
[nit] I'm not sure how I feel about lowercase true. It's definitely mentioned as acceptable but it's completely outside the convention of CMake. (See https://cmake.org/cmake/help/latest/command/if.html)